src/App.tsx

```diff
@@ -1,24 +1,247 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect, useRef } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import * as postServices from './api/todos';
+import { Todos } from './api/components/todos';
+import { Footer } from './api/components/footer';
+import { Error } from './api/components/error';
+import { Loader } from './api/components/loader';
 
-const USER_ID = 0;
+const USER_ID = 12030;
+
+function getPreparedTodos(todos: Todo[], filter: string): Todo[] {
+  const preparedTodos = todos.filter(todo => {
+    switch (filter) {
+      case 'completed':
+        return todo.completed;
+
+      case 'active':
+        return !todo.completed;
+
+      default:
+        return true;
+    }
+  });
+
+  return preparedTodos;
+}
+
+// function error(er: string) {
+//   switch (er) {
+//     case 'load':
+//       return 'Unable to load todos';
+//     case 'empty':
+//       return 'Title should not be empty';
+//     case 'add':
+//       return 'Unable to add a todo';
+//     case 'delete':
+//       return 'Unable to delete a todo';
+//     case 'update':
+//       return 'Unable to update a todo';
+//     default: return '';
+//   }
+// }
+
+function todosCounter(todos: Todo[]) {
+  return todos.filter(todo => !todo.completed).length;
+}
 
 export const App: React.FC = () => {
+  const [loading, setLoading] = useState(false);
```

Comment:
Rename all boolean variables according to this [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)
Fix all cases
```suggestion
  const [isLoading, setIsLoading] = useState(false);
```